src/openssl.c

```diff
@@ -1101,6 +1101,9 @@ void _libssh2_openssl_crypto_init(void)
     ENGINE_register_all_complete();
 #endif
 #endif
+#ifdef DEBUG_WOLFSSL
```

Comment:
Let's also guard it with `LIBSSH2_WOLFSSL`.

Reply:
Added `LIBSSH2_WOLFSSL` to macro check.

Reply:
Thanks @julek-wolfssl! and merging.